--- cargo_toml-0.13.3/Cargo.toml	1970-01-01T00:00:01+00:00
+++ cargo_toml-0.13.3/Cargo.toml	2023-03-24T13:12:44.543818+00:00
@@ -14,7 +14,7 @@
 name = "cargo_toml"
 version = "0.13.3"
 authors = ["Kornel <kornel@geekhood.net>"]
-exclude = ["tests"]
+exclude = ["tests", "the-real-tom.jpeg"]
 description = "`Cargo.toml` struct definitions for parsing with Serde"
 homepage = "https://lib.rs/cargo_toml"
 documentation = "https://docs.rs/cargo_toml"
