--- linux-loader-0.8.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ linux-loader-0.8.1/Cargo.toml	2023-03-24T13:34:28.241470+00:00
@@ -21,6 +21,7 @@
 keywords = ["kernel"]
 license = "Apache-2.0 AND BSD-3-Clause"
 repository = "https://github.com/rust-vmm/linux-loader"
+exclude = ["/CODEOWNERS", "/coverage_config*"]
 
 [profile.bench]
 lto = true
@@ -36,9 +37,6 @@
 [dependencies.vm-memory]
 version = "0.10.0"
 
-[dev-dependencies.criterion]
-version = "0.3.5"
-
 [dev-dependencies.vm-memory]
 version = "0.10.0"
 features = ["backend-mmap"]
